[2022-06-09 16:45:14] [INFO] [Recovery] 2022/06/09 - 16:45:14 panic recovered:
Error 1040: Too many connections
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/errorx/errorx.go:40 (0x1d8b95a)
	Dangerous: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/ginx/errorx.go:10 (0x1dbf909)
	Dangerous: }
C:/Users/mensyli1/work/workspace/middleware/src/controller/postgres/postgres.go:42 (0x1dbf8f8)
	GetById: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1dc3027)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/jwt.go:19 (0x1dc300e)
	Auth.func1: 
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1dc314b)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/logrus.go:62 (0x1dc3128)
	Logrus.func1: //ÁªìÊùüÊó∂Èó¥
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1dc20ba)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/recovery.go:94 (0x1dc20a6)
	CustomRecoveryWithWriter.func1: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1d651fd)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489 (0x1d64e85)
	(*Engine).handleHTTPRequest: c.writermem.WriteHeaderNow()
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445 (0x1d649e4)
	(*Engine).ServeHTTP: 
C:/install/Dev/Env/Go/src/net/http/server.go:2879 (0x10c459a)
	serverHandler.ServeHTTP: }
C:/install/Dev/Env/Go/src/net/http/server.go:1930 (0x10bfc47)
	(*conn).serve: w.cancelCtx()
C:/install/Dev/Env/Go/src/runtime/asm_amd64.s:1581 (0xe14b00)
	goexit: CALL	runtime¬∑goexit1(SB)	// does not return
[0m (github.com/mensylisir/kmpp-middleware/src/util/aop.CustomRecoveryWithWriter.func1.1: 82) {map[]} 
[2022-06-09 16:45:18] [INFO] [Recovery] 2022/06/09 - 16:45:18 panic recovered:
Error 1040: Too many connections
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/errorx/errorx.go:40 (0x1d8b95a)
	Dangerous: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/toolkits/pkg@v1.2.9/ginx/errorx.go:10 (0x1dbf909)
	Dangerous: }
C:/Users/mensyli1/work/workspace/middleware/src/controller/postgres/postgres.go:42 (0x1dbf8f8)
	GetById: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1dc3027)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/jwt.go:19 (0x1dc300e)
	Auth.func1: 
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1dc314b)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/logrus.go:62 (0x1dc3128)
	Logrus.func1: //ÁªìÊùüÊó∂Èó¥
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1dc20ba)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/middleware/src/util/aop/recovery.go:94 (0x1dc20a6)
	CustomRecoveryWithWriter.func1: }
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/context.go:165 (0x1d651fd)
	(*Context).Next: c.index++
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:489 (0x1d64e85)
	(*Engine).handleHTTPRequest: c.writermem.WriteHeaderNow()
C:/Users/mensyli1/work/workspace/mwpath/pkg/mod/github.com/gin-gonic/gin@v1.7.4/gin.go:445 (0x1d649e4)
	(*Engine).ServeHTTP: 
C:/install/Dev/Env/Go/src/net/http/server.go:2879 (0x10c459a)
	serverHandler.ServeHTTP: }
C:/install/Dev/Env/Go/src/net/http/server.go:1930 (0x10bfc47)
	(*conn).serve: w.cancelCtx()
C:/install/Dev/Env/Go/src/runtime/asm_amd64.s:1581 (0xe14b00)
	goexit: CALL	runtime¬∑goexit1(SB)	// does not return
[0m (github.com/mensylisir/kmpp-middleware/src/util/aop.CustomRecoveryWithWriter.func1.1: 82) {map[]} 
